src/work-allocation/resolvers/task.resolver.ts

```diff
@@ -1,25 +1,30 @@
 import { Injectable } from '@angular/core';
 import { ActivatedRouteSnapshot, Resolve, Router, RouterStateSnapshot } from '@angular/router';
-import { EMPTY, Observable } from 'rxjs';
+import { combineLatest, EMPTY, Observable } from 'rxjs';
 import { catchError } from 'rxjs/operators';
+import { Caseworker } from '../models/dtos';
 
 import { Task } from '../models/tasks';
-import { WorkAllocationTaskService } from '../services';
+import { CaseworkerDataService, WorkAllocationTaskService } from '../services';
 import { handleFatalErrors, WILDCARD_SERVICE_DOWN } from '../utils';
 
 @Injectable({ providedIn: 'root' })
-export class TaskResolver implements Resolve<Task> {
+export class TaskResolver implements Resolve<any> {
   constructor(
     private readonly service: WorkAllocationTaskService,
-    private readonly router: Router
+    private readonly router: Router,
+    private readonly caseworkerService: CaseworkerDataService
   ) {}
 
-  public resolve(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): Observable<Task> {
-    return this.service.getTask(route.paramMap.get('taskId')).pipe(
+  public resolve(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): Observable<any> {
```

Comment:
Can you get this to return something more meaningful than `any`? It _looks_ like it should be an array along the lines of `[ Task, Caseworker[] ]`...?

Reply:
Or you could pipe it into an object of the form `{ task, caseworkers }` rather than an array. 🤷‍♂️

Reply:
Again agree, forkJoin. supports object notation. combineLatest supports []. Will do the change

Reply:
Done